### PR TITLE
Quick read of 00

### PR DESCRIPTION
**Pull request by cabo**
A couple of small notes (from typos to todo/crefs).  Use as convenient.

### CODE REVIEW

File: draft-ietf-core-dns-over-coap.md [hunk 1]
```diff
@@ -75,9 +72,9 @@ To prevent resource requirements of DTLS or TLS on top of UDP (e.g.,
 introduced by DNS over QUIC {{?RFC9250}}), DoC allows
 for lightweight end-to-end payload encryption based on OSCORE.
 
-~~~ drawing
+~~~ aasvg
```
Comment (miri64) [inline draft-ietf-core-dns-over-coap.md:75]: Is there some `aasvg` way to have the dotted/dashed arrow on the right-hand side of the graphic parsed correctly?

File: draft-ietf-core-dns-over-coap.md [hunk 2]
```diff
@@ -132,19 +129,24 @@ Basic Message Exchange
...
+messages that use the CoAP FETCH {{!RFC8132}} method.
 Requests SHOULD include an Accept option to indicate the type of content that can be parsed in the response.
 
-The CoAP request SHOULD be carried in a Confirmable (CON) message, if the transport used does not provide reliable message exchange.
+The CoAP request SHOULD be carried in a Confirmable (CON) message, if the transport used does not provide reliable message exchange.[^layer-violation]
+
+{: source=" -- cabo"}
+[^layer-violation]: I'm not sure we need this layer violation here; if
+    the client doesn't really need the response, why require it?
+    Also: What are the cases where the "SHoULD" can be ignored?
```
Comment (miri64) [inline draft-ietf-core-dns-over-coap.md:149]: Mostly, we wanted to say: "If needed, we want to re-use the retransmission mechanism of CoAP".

Not being interested in the response might not make this much sense in a DNS use case.

File: draft-ietf-core-dns-over-coap.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-core-dns-over-coap.md:75]: It looked nice enough for me.
I'm not aware about dashed/dotted line support in aasvg.
Not sure my attempt at a8c912e is an improvement.

Comment (miri64) [reply draft-ietf-core-dns-over-coap.md:75]: IMHO it does. At least the arrow tips match now and the line isn't weirdly misaligned.

File: draft-ietf-core-dns-over-coap.md (hunk 2 above)
Comment (miri64) [reply draft-ietf-core-dns-over-coap.md:149]: > (A statement of fact that the reliability of the message layer, e.g., CON, is used to replace the retransmissions of the DNS protocol, would fit well.)

I can do that as a follow-up.

**Review by miri64 — APPROVED** [review]

**Pull request merged**